geometry_msgs/Point object_pos
---
geometry_msgs/Point point1
geometry_msgs/Point point2
